### PR TITLE
DB/World: pandaren action bar

### PR DESCRIPTION
**Pull request by zTerragor**
DB/World: Fix missing 'Quaking Palm' pandaren mage action bar

### CODE REVIEW

File: sql/updates/master/world/world_2023_11_10_13_33.sql [hunk 1]
```diff
@@ -0,0 +1,8 @@
+DELETE FROM `playercreateinfo_spell` WHERE `racemask`=8388608 AND `classmask`=64 AND `Spell`=403;
```
Comment (leelf00) [inline sql/updates/master/world/world_2023_11_10_13_33.sql:1]: need check

**Review by leelf00 — CHANGES_REQUESTED** [review]
Where is this file from?

**Comment by jasongdove** [comment]
@zTerragor Pandaren action bars for all classes were fixed with https://github.com/Legends-of-Azeroth/Legends-of-Azeroth-Pandaria-5.4.8/pull/268 - please confirm.

**Comment by leelf00** [comment]
Fixed by https://github.com/Legends-of-Azeroth/Legends-of-Azeroth-Pandaria-5.4.8/pull/268

**Pull request closed without merging**